doc-examples/jdbc-example-kotlin/src/main/kotlin/example/BookRepository.kt

```diff
@@ -124,6 +125,17 @@ interface BookRepository : CrudRepository<Book, Long> { // <2>
     fun calculateSum(bookId: @NonNull Long): Long
     // end::procedure[]
 
+    // tag::onetomanycustom[]
+    @Query("""
+        SELECT b.*,
+               r.id AS reviews_id, r.reviewer AS reviews_reviewer, r.content AS reviews_content, r.book_id AS reviews_book_id
+        FROM book b INNER JOIN review r ON b.id = r.book_id
+        WHERE b.title = :title
+        """)
+    @Join("reviews")
```

Comment:
how does one customise the separator? Looks like it defaults to underscore `_`? Can we document that?

Reply:
Added note about that.